batchflow/models/torch/blocks.py

```diff
@@ -298,3 +298,84 @@ def __init__(self, inputs=None, layout='nacd', filters=None, kernel_size=3, stri
     def forward(self, x):
         x = super().forward(x)
         return x if self.skip else x[:, self.input_num_channels:]
+
+
+class ResNeStBlock(ConvBlock):
+    """ ResNeSt Module: pass tensor through one or multiple (`n_reps`) blocks, each of which is a
```

Comment:
Make a clear description of what happens to the `input` tensor:

- first of all, we split it into `radix` groups
- then ...
- then apply `sac`
- then ...